assets/sass/patterns/atoms/list.scss

```diff
@@ -18,3 +18,7 @@ $prefixes: ("alpha-lower": "lower-latin", "alpha-upper": "upper-latin", "bullet"
     }
   }
 }
+
+.list--teaser {
+  @include padding($blg-space-small-in-px, "top");
```

Comment:
@davidcmoulton there is already a `margin-bottom` of 23px (1.4375rem) on `.teaser`. If I wrap the text in a `<p>` and don't add a list then the only visible difference is caused by the `margin-botton` on the `<p>` of 24px (1.5rem).

I was only considering adding the `<p>` tag for annual-reports and they will all have a list underneath the impact statement now.